src/custom/abis/types/VCow.d.ts

```diff
@@ -31,6 +32,7 @@ interface VCowInterface extends ethers.utils.Interface {
     "nativeTokenPrice()": FunctionFragment;
     "swappableBalanceOf(address)": FunctionFragment;
     "balanceOf(address)": FunctionFragment;
+    "swapAll()": FunctionFragment;
```

Comment:
nenad, this is the 3rd PR with this change. Please, set the base branches correctly so this changes are review only once and don't make us miss the changes in this PR

Reply:
Yep, I've fixed this and its not in the PR anymore :+1: